app/models/article.rb

```diff
@@ -854,7 +884,7 @@ def set_cached_entities
   end
 
   def set_all_dates
-    set_published_date
+    # set_published_date
```

Comment:
What are your thoughts on removing commented code?  To me commented out code is hard to know if it's removal was intentional or not.

Reply:
I think it should be removed :)